### PR TITLE
feat: enable moonraker object preprocessing

### PR DESCRIPTION
**Pull request by matmen**
In preparation for https://github.com/fluidd-core/fluidd/issues/359

### CODE REVIEW

File: src/modules/moonraker/filesystem/home/pi/klipper_config/moonraker.conf [hunk 1]
```diff
@@ -6,6 +6,7 @@ enable_debug_logging: False
 [file_manager]
 log_path: ~/klipper_logs
 config_path: ~/klipper_config
+enable_object_processing: True
```
Comment (pedrolamas) [inline src/modules/moonraker/filesystem/home/pi/klipper_config/moonraker.conf:9]: We might want to add a note here like MainsailOS has:

https://github.com/mainsail-crew/MainsailOS/blob/develop/src/modules/moonraker/filesystem/home/pi/klipper_config/moonraker.conf#L12

**Review by pedrolamas — APPROVED** [review]

**Review by pedrolamas — APPROVED** [review]

**Comment by pedrolamas** [comment]
I've had a bit of second thoughts on this... wouldn't it make more sense to default the `enable_object_processing` to `False` and allow more advanced users to manually change it?

**Comment by matmen** [comment]
I think having it enabled by default makes sense for optimal user experience (no need for users to add a slicer post-processing script or manually toggle it on for object cancellation). What's the reason for having it disabled by default? And if it was disabled by default, we should probably add a docs page explaining the methods to enable object cancellation (and maybe even have gcode-viewer auto-loading turned off?).

**Comment by pedrolamas** [comment]
I'm just a bit mostly worried on the performance impact of users having this enabled and installing FluiddPi on a older Raspberry Pi, hence why I had second thoughts on this!

**Comment by matmen** [comment]
Fair. I honestly have no idea what the share of older (/ low performance) devices running FluiddPi looks like, and I don't know what the performance impact is on those. I may be able to test the performance impact (at least on a Pi 3, my Pi zero doesn't have WiFi), but I think having it opt-in is just fine too.
What's your opinion on making auto loading in the gcode viewer-opt in too?

**Comment by pedrolamas** [comment]
> What's your opinion on making auto loading in the gcode viewer-opt in too?

Yes, that makes sense to me!

**Pull request merged**